pkg/services/alerting/engine.go

```diff
@@ -22,12 +22,15 @@ type AlertingService struct {
 
 	execQueue chan *Job
 	//clock         clock.Clock
-	ticker        *Ticker
-	scheduler     Scheduler
-	evalHandler   EvalHandler
-	ruleReader    RuleReader
-	log           log.Logger
-	resultHandler ResultHandler
+	ticker                             *Ticker
+	scheduler                          Scheduler
+	evalHandler                        EvalHandler
+	ruleReader                         RuleReader
+	log                                log.Logger
+	resultHandler                      ResultHandler
+	alertingEvaluationTimeoutSeconds   int
```

Comment:
Don't exactly why these settings needs to be part of the `AlertingService`?

Reply:
I think it would be better if there were separate configuration object, but it is not convenient to add

Reply:
We're migrating away from using the global settings and use settings.Cfg instead, but I reckoned this is something to fix in the future since all alerting settings should be in settings.Cfg and not global settings then.